readme.md

```diff
@@ -43,12 +43,58 @@
 
 ## Test
 
-- `./eng/Generate.ps1` (at root in PowerShell Core)
-- `dotnet test` (at root)
+**`./eng/Generate.ps1` (at root in PowerShell Core)**
+
+This tests changes across many swagger definitions and samples.
+
+These arguments change the behavior:
+- -fast option skips Swagger -> YAML IL step. Much faster when only making codegen changes
+- -fast SWAGGER_NAME to run only one case
+
+One important note is that Generate.ps1 runs logic similar to /t:GenerateCode but does not use it directly 
```

Comment:
Could you expand a bit on what this means?

Reply:
@pakrym - Can you comment, this came directly from my notes in talking to you. Is it worth keeping?

Reply:
I don't think I would tell the general population much. We can move it to internals doc.

Reply:
Done.